lib/models/App.js

```diff
@@ -335,6 +336,44 @@ class App {
     return this
   }
 
+  loadPlugins() {
+    if (!this.config) {
+      this.loadConfig()
+    }
+
+    const dirPath = path.resolve(
+      this.projectDir(),
+      this.config.app.directory,
+      'plugins'
+    )
+
+    /* if application provide the same plugin key,
+     * it will override the sl-express one
+     */
+
+    const plugins = AppHelper.importPlugins(
+      dirPath,
+      { required: false, only: this.config.app.plugins }
+    )
+
+    const existingKeys = Object.keys(plugins)
+    const remainingKeys = this.config.app.plugins.filter( (key) => {
```

Comment:
```suggestion
    const remainingKeys = this.config.app.plugins.filter((key) => {
```

Reply:
i think we need a linter lol

Reply:
sure. but before a linter, we need somebody to have time for fixing. a bit low priority at the moment. sad

Reply:
let me fix this by the way. I tested the code in ben.shopline.com project. let me find out why

Reply:
err... I used to use () even it works without ()